src/core/public/loading_count/__snapshots__/loading_count_service.test.ts.snap

```diff
@@ -1,38 +0,0 @@
-// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
**optional nit**  IMHO it's crucial to keep git story clean in a big project because often we need to find when and why something was added in the codebase. so I'd use `git mv` here

Reply:
@restrry I totally agree. I think this is a limitation in the github PR changes view. The commit (https://github.com/elastic/kibana/pull/31800/commits/5f171bd727f12440eb2e94ceb3b1e4fbfe3e6d2f) shows the rename, but because a subsequent commit rewrote so much of the file, the summary view makes it appear as if a new file was created.